### PR TITLE
feature : added a Toast component

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+.primary-btn{
+    background-color: var(--warning-color);
+  padding: 8px 16px;
+  margin: 1rem;
+  font-size: large;
+  color: white;
+  border: none;
+  font-weight: bold;
+  border-style: none;
```
Comment (kiranpratyush) [inline Badge/badge.css:120]: We can remove this

File: CSS/card.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      flex-direction: row;
+      height: 1.9rem;
+      width: 1.9rem;
+      border: 1rem;
+    }
+    .card-icons i:hover{
+      background-color:#f1f1f3 ;
```
Comment (kiranpratyush) [inline CSS/card.css:94]: Can use variable here

**Review by kiranpratyush — APPROVED** [review]
We can format our code to look beautiful

**Review by shagunbagrecha — APPROVED** [review]

**Review by MiheerTamkhane — APPROVED** [review]
Create assets folder and add images, svg in it.

**Pull request merged**